docs/snippets/all/archetypes/geo_line_string_simple.rs

```diff
@@ -0,0 +1,20 @@
+//! Log a simple geospatial line string.
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let rec = rerun::RecordingStreamBuilder::new("rerun_example_geo_line_strings").spawn()?;
+
+    rec.log(
+        "colorado",
+        &rerun::GeoLineStrings::new([[
+            [41.0000, -109.0452],
```

Comment:
In none of the languages is it clear which order latitude/longitude comes in, which is a big foot-gun [since apparently none agrees on the correct order](https://stackoverflow.com/questions/7309121/preferred-order-of-writing-latitude-longitude-tuples-in-gis-services)

I think we should consider a design where this is explicit, e.g.

```suggestion
        &rerun::GeoLineStrings::from_lat_long([[
            [41.0000, -109.0452],
```